cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -123,6 +123,22 @@ $ virtctl image-upload ... image.raw.gz
 Restarting a virtual machine while it is in the *Importing* state results in an error.
 https://bugzilla.redhat.com/show_bug.cgi?id=1673921[(BZ#1673921)]
 
+* If the kubelet on a node crashes or restarts, this causes the kubelet to
+incorrectly report 0 KVM devices. Virtual machines are not properly scheduled
+on affected nodes.
++
+Verify the number of devices that the kubelet reports by running
+`$ oc get node $NODE | grep devices.kubevirt`. The output on an affected node
+shows `devices.kubevirt.io/kvm: 0`.
+https://bugzilla.redhat.com/show_bug.cgi?id=1681175[(BZ#1681175)]
```

Comment:
We tend to format BZ #s for bugs and known issues as:

    (link:https://bugzilla.redhat.com/show_bug.cgi?id=1681175[*BZ#1681175*])

Reply:
Thanks for the heads up! I changed the link formatting throughout the Release Notes.